Umweltzone/src/main/res/values-cs-rCZ/strings.xml

```diff
@@ -0,0 +1,98 @@
+    <string name="city_info_provision_applies_to_vehicles_fragment_cars_and_hgvs">auta a těžká nákladní vozidla</string>
+    <string name="city_info_provision_applies_to_vehicles_fragment_hgvs">těžká nákladní vozidla</string>
+    <string name="city_info_provision_applies_to_vehicles_fragment_cars">auta</string>
+    <string name="city_info_geometry_updated_at_date_format">MMM d, yyyy</string>
+    <string name="city_info_list_of_cities">Tato oblast obsahuje následující města nebo městské části:</string>
+    <string name="map_my_location_activation_hint">Klepnutím zde k nastavte vyhledávání polohy, aby se vaše poloha mohla zobrazit na mapě.</string>
+    <string name="appinfo_application_logo_description">Logo  &#8220;Umweltzone&#8221; aplikace</string>
+    <string name="appinfo_subtitle">Android aplikace pro nízkoemisní oblasti (německy: Umweltzonen) a oblasti se zákazem dieselů v Německu</string>
+    <string name="appinfo_copyright_content">"    (&#169;) Tobias Preuss, 2013 &#8211; 2020"</string>
+    <string name="city_info_zone_number_as_of">Od <xliff:g example="Oct 1, 2099" id="date">%1$s</xliff:g> pouze vozidla <xliff:g example="with a green" id="with_colors">%2$s</xliff:g>emisní plaketou smí vjet do nízkoemisní zóny.</string>
+    <string name="city_info_zone_number_since">Od <xliff:g example="Oct 1, 2099" id="date">%1$s</xliff:g> pouze vozidla <xliff:g example="with a green" id="with_colors">%2$s</xliff:g>emisní plaketou smí vjet do nízkoemisní zóny.</string>
+    <string name="city_info_zone_number_since_text_fragment_green">se zelenou</string>
+    <string name="city_info_zone_number_since_text_fragment_yellow">se žlutou nebo s zelenou</string>
+    <string name="city_info_zone_number_since_text_fragment_red">s červenou, žlutou nebo se zelenou</string>
+    <string name="city_info_zone_number_none">Do nízkoemisní zóny mohou vjet vozidla všech emisních tříd.</string>
+    <string name="city_info_next_zone_number_as_of">Od  <xliff:g example="Feb 1, 2016" id="date">%1$s</xliff:g> pouze vozidla <xliff:g example="with a green" id="with_colors">%2$s</xliff:g> emisní plaketou smí vjet do nízkoemisní zóny.</string>
+    <string name="appinfo_rating_url_title">Umweltzone v Obchodě Google Play</string>
+    <string name="appinfo_rating_content">Rádi obdržíme vaše pozitivní hodnocení v Obchodě Google Play.</string>
+    <string name="appinfo_license_url_title_creative_commons">Licence Creative Common</string>
+    <string name="zone_not_drawable_title">Pomozte nyní.</string>
+    <string name="city_info_is_congruent_with_lez">Tato oblast se shoduje s nízkoemisní zónou.</string>
+    <string name="appinfo_sourcecode_content">Zdrojový kód aplikace je přístupný online.</string>
+    <string name="city_info_provision_applies_to_vehicles">"Toto opatření se vztahuje na <xliff:g example="cars and heavy goods vehicles" id="vehicles">%1$s</xliff:g>.   "</string>
```

Comment:
Please remove the outer double quotes and the whitespace. They cause formatting issues. It might be that the editor added them without your knowledge.